services/frontend-service/src/ui/components/dropdown/dropdown.tsx

```diff
@@ -0,0 +1,118 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with kuberpult.  If not, see <http://www.gnu.org/licenses/>.
+
+Copyright 2021 freiheit.com*/
+import { Checkbox, FormControl, InputLabel, MenuItem, Select } from '@material-ui/core';
+import classNames from 'classnames';
+import { useCallback, useRef } from 'react';
+import { useTeamNames } from '../../utils/store';
+import { useSearchParams } from 'react-router-dom';
+
+export type DropdownProps = {
+    className?: string;
+    floatingLabel?: string;
+    leadingIcon?: string;
+};
+
+export const DropdownSelect: React.FC<{
+    handleChange: (event: any) => void;
+    isEmpty: (arr: string[] | undefined) => boolean;
+    floatingLabel: string | undefined;
+    teams: string[];
+    selectedTeams: string[];
+}> = (props) => {
+    const { handleChange, isEmpty, floatingLabel, teams, selectedTeams } = props;
+    const renderValue = useCallback((selected: string[]) => selected.join(', '), []);
+
+    return (
+        <FormControl variant="outlined" fullWidth data-testid="teams-dropdown-formcontrol">
+            <InputLabel
+                htmlFor="teams"
+                id="teams"
+                shrink={!isEmpty(selectedTeams) ? true : false}
```

Comment:
The ternary expression is unnecessary. `!isEmpty(selectedTeams)` is enough.

Reply:
I swear this made sense before the isEmpty function existed hahah
Fixed!